core/jazz_es-kinesis-log-streamer/index.js

```diff
@@ -14,7 +14,7 @@
 // limitations under the License.
 // =========================================================================
 
-var https = require('https');
+var http = require('http');
```

Comment:
Can we use [request](https://github.com/request/request) module instead of this http <-> https switch? `request` module should take care of this for us. Covers use case when jazz admin replaces es_endpoint with https or http endpoint value..